monix-tail/shared/src/test/scala/monix/tail/IterantBasicSuite.scala

```diff
@@ -59,6 +59,20 @@ object IterantBasicSuite extends BaseTestSuite {
     assertEquals(f.value, Some(Success(1)))
   }
 
+  test("Iterant.liftF") { implicit s =>
```

Comment:
We probably need a test using ScalaCheck too.

```scala
test("any F can be lifted) { implicit s =>
  check1 { (fa: IO[Int]) =>
    Iterant[IO].liftF(fa).headOptionL <-> fa.map(Some.apply)
  }
}
```

Or something along those lines.